src/mocks.ts

```diff
@@ -0,0 +1,19 @@
+import consola from 'consola'
+
+export function mockConsola () {
+  const mock = consola
```

Comment:
Is there any reason to rename consola to mock?

Reply:
To completely mock, it does not work using the global object `consola`

Reply:
They are same reference....

Reply:
You're right :satisfied: